lib/postcode_validation/use_case/validate_address.rb

```diff
@@ -1,5 +1,6 @@
 module PostcodeValidation
   module UseCase
+    require_relative 'format_validator'
     class ValidateAddress
       def initialize(address_match_gateway:, logger: nil)
```

Comment:
maybe `@format_validator = FormatValidator.new`

(Then ofc the "build" method would no longer be a self. method)

I think assigning an instance variable in the constructor like this keeps the door open if we ever want to invert this dependency.